csharp/src/Microsoft.ML.OnnxRuntime/Tensors/ArrayUtilities.shared.cs

```diff
@@ -256,5 +256,18 @@ public static int TransformIndexByStrides(int index, int[] sourceStrides, bool s
 
             return transformIndex;
         }
+
+        public static T[] GetEmpty<T>()
```

Comment:
Given `EmptyArray<T>.Value` is readonly, should this be returning a `readonly T[]`?

Reply:
There is no such thing as a `readonly T[]` in .NET

The closest is `ReadOnlySpan<T>`, but that is altogether different and not always usable in the same scenarios. It being empty means that there are no mutations you can make to it or its contents.